mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -249,8 +249,8 @@ static Value createLinalgBodyCalculationForElementwiseOp(
     auto shifted =
         rewriter.create<arith::ShRSIOp>(loc, resultTypes, args[0], subtract)
             ->getResults();
-    auto truncated =
-        rewriter.create<arith::TruncIOp>(loc, i1Ty, shifted, std::nullopt);
+    auto truncated = rewriter.create<arith::TruncIOp>(
+        loc, i1Ty, shifted, llvm::ArrayRef<NamedAttribute>());
```

Comment:
If I understand correctly, you can't use `{}` in calls to `create<>` because the arguments get passed on with `std::forward` which does not work with initializer lists - but I'm not an expert.